package.json

```
@@ -33,8 +33,8 @@
     "test:puppeteer": "jest --ci --env=jsdom --colors ./puppeteer",
     "test:unit": "test -z $CI && jest --env=jsdom --coverage --colors --testPathIgnorePatterns=\"src/integration\" \"puppeteer\" || jest --ci --runInBand --env=jsdom --coverage --colors --testPathIgnorePatterns=\"src/integration\" \"puppeteer\"",
     "test:unit:watch": "npm run test:unit -- --watch",
-    "test:integration:ci": "jest --ci --runInBand --colors ./src/integration/pages/**/*.test.js",
-    "test:integration": "./scripts/killApp.sh && npm run build:local && (npm run start & wait-on -t 20000 http://localhost:7080/status) && jest --runInBand --colors ./src/integration/pages/**/*.test.js",
+    "test:integration": "./scripts/killApp.sh && npm run build:local && (npm run start & wait-on -t 20000 http://localhost:7080/status) && jest --no-cache --runInBand --colors ./src/integration/pages/**/*",
```

Perhaps not in scope for this PR but AFAIK we still have the issue where a zombie simorgh app is left running once the integration tests have finished, I've not tried this suggestion but I'm fairly confident it'll fix it.
```suggestion
    "test:integration": "./scripts/killApp.sh && npm run build:local && (npm run start & wait-on -t 20000 http://localhost:7080/status) && (jest --no-cache --runInBand --colors ./src/integration/pages/**/*) && ./scripts/killApp.sh",
```

---

@karinathomasbbc I've been meaning to chat to you about how to solve this

---

I'm happy to make a separate PR with these changes @andrewscfc. Thanks